portfolio/src/main/webapp/script.js

```diff
@@ -137,15 +137,54 @@ function getMessage() {
 function handleResponse(response) {
   const textPromise = response.text();
   
-  textPromise.then(addMessageToDom);
+  textPromise.then(addSingleMessageToDom);
 }
 
 /**
- * Adds the given message to the DOM.
+ * Adds the given single message to the DOM.
  * @param {string} message The text to be added inside of the div 
        message-container. 
  */
-function addMessageToDom(message) {
+function addSingleMessageToDom(message) {
   const messageContainer = document.getElementById('message-container');
   messageContainer.innerHTML = message;
 }
+
+/**
+ * Adds multiple messages to the DOM as list elements.
```

Comment:
Here too, can be made more simple (can remove the @param):

Adds {@code messagesList} to the DOM as list elements.

Reply:
got it
done!

Reply:
I also simplified the method headers for addSingleMessageToDom() and appendTextToList() to use @code instead of @param.